src/index.js

```diff
@@ -95,48 +111,164 @@ class Masq {
     })
   }
 
+  /**
+   * If this is the first time, this.dbs.profiles is empty.
+   * We need to get masq-profiles hyperdb key of masq.
+   */
+
   requestMasqAccess () {
     // Subscribe to channel for a limited time to sync with masq
-    this.hub = signalhub(this.channel, ['localhost:8080'])
+    const hub = signalhub(this.channel, [HUB_URL])
+    let sw = null
+
+    if (swarm.WEBRTC_SUPPORT) {
+      sw = swarm(hub)
+    } else {
+      sw = swarm(hub, { wrtc: require('wrtc') })
+    }
+
+    sw.on('peer', (peer, id) => {
+      peer.on('data', data => _handleData(data, peer))
+    })
+
+    sw.on('close', () => {
+      hub.close()
+    })
+
+    sw.on('disconnect', (peer, id) => {
+      sw.close()
+      hub.close()
```

Comment:
Should not be needed as the sw.on('close') event will close the hub too

Reply:
done